app/src/main/java/org/glucosio/android/db/User.java

```diff
@@ -40,14 +40,14 @@
     @Required
     private String preferred_unit_weight;
     private String preferred_range;
-    private int custom_range_min;
-    private int custom_range_max;
+    private double custom_range_min;
+    private double custom_range_max;
 
     public User() {
 
     }
 
-    public User(int id, String name, String preferred_language, String country, int age, String gender, int dType, String pUnit, String a1cUnit, String weightUnit, String pRange, int minRange, int maxRange) {
+    public User(int id, String name, String preferred_language, String country, int age, String gender, int dType, String pUnit, String a1cUnit, String weightUnit, String pRange, double minRange, double maxRange) {
```

Comment:
Make this private and implement a builder?

Reply:
Added builder